packages/react-router-redux/modules/ConnectedRouter.js

```diff
@@ -26,7 +26,9 @@ class ConnectedRouter extends Component {
               payload: location
             })
 
-            return children
+            return !Array.isArray(children) || children.length ? ( // Preact defaults to empty children array
+              React.Children.only(children)
+            ) : null
```

Comment:
Well, render can throw, so that's a valid thing. Also, it would cover both the initial children and if those children ever changed in the future (though, that's pretty uncommon). But if we apply the same behavior as `<Router>`, that's probably a good thing overall.